src/I18n/Time.php

```diff
@@ -856,10 +856,10 @@ public static function parseTime($time, $format = null)
     /**
      * Convenience method for getting the remaining time from a given time.
      *
-     * @param DateTime $datetime The date to get the remaining time from.
-     * @return DateInterval|bool The DateInterval object representing the difference between the two dates or FALSE on failure.
+     * @param \DateTime $datetime The date to get the remaining time from.
+     * @return \DateInterval|bool The DateInterval object representing the difference between the two dates or FALSE on failure.
      */
-    public static function fromNow($datetime)
+    public static function fromNow(\DateTime $datetime)
```

Comment:
Is that the projects standard? There are a few other occurrences of non imported hints, I could fix this all in one go then.